ui/app/components/ui/popover/popover.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import PopOverheader from './popover.header.component'
+
+const PopOver = ({ title, children, onClose }) => (
+  <div className="popover-wrap">
+    <div className="popover-content">
+      <PopOverheader title={title} onClose={onClose} />
+      {children}
+    </div>
+    <a href="#" className="popover-bg" onClick={onClose} />
+  </div>
+)
+
+PopOver.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
```

Comment:
This PropType is redundant because an array of nodes is also a node.

I noticed this pattern already existed in the codebase, so I've removed it in #8126